PWGHF/D2H/DataModel/ReducedDataModel.h

```diff
@@ -370,6 +370,35 @@ DECLARE_SOA_TABLE(HfCandBpConfigs, "AOD", "HFCANDBPCONFIG", //! Table with confi
                   hf_cand_bplus_config::MySelectionFlagD0,
                   hf_cand_bplus_config::MySelectionFlagD0bar,
                   hf_cand_bplus_config::MyInvMassWindowD0Pi);
+
+// Charm resonances analysis
+namespace hf_reso_cand_reduced
+{
+DECLARE_SOA_COLUMN(InvMass, invMass, float); //! Invariant mass of V0/D candidate in GeV/c2
+DECLARE_SOA_COLUMN(Px, px, float);           //! Momentum of V0/D candidate in GeV/c
+DECLARE_SOA_COLUMN(Py, py, float);
+DECLARE_SOA_COLUMN(Pz, pz, float);
+} // namespace hf_reso_cand_reduced
+
+DECLARE_SOA_TABLE(HfRedVzeros, "AOD", "HFREDVZERO", //! Table with V0 candidate information for resonances reduced workflow
+                  o2::soa::Index<>,
+                  hf_track_index_reduced::Prong0Id, hf_track_index_reduced::Prong1Id,
+                  hf_track_index_reduced::HfRedCollisionId,
+                  hf_cand::XSecondaryVertex, hf_cand::YSecondaryVertex, hf_cand::ZSecondaryVertex,
+                  hf_reso_cand_reduced::InvMass,
+                  hf_reso_cand_reduced::Px,
+                  hf_reso_cand_reduced::Py,
+                  hf_reso_cand_reduced::Pz);
+
+DECLARE_SOA_TABLE(HfRedD, "AOD", "HFREDD", //! Table with D+/D* candidate information for resonances reduced workflow
```

Comment:
I would suggest a different name (also considering that, if we add the D0, it will be likely a separate table)
```suggestion
DECLARE_SOA_TABLE(HfRed3PrNoTrks, "AOD", "HFRED3PRNOTRK", //! Table with D+/D* candidate information for resonances reduced workflow
```